scripts/java-api-check.sh

```diff
@@ -50,27 +55,30 @@ PREVIOUS_RELEASE_DIR=$(dirname "${PREVIOUS_RELEASE}")
 
 LAST_STABLE_VERSION_TAG=$4 #v10.3.0
 LAST_STABLE_VERSION_TAG_ARRAY=($LAST_STABLE_VERSION_TAG)
-LAST_STABLE_VERSION=${LAST_STABLE_VERSION_TAG_ARRAY[0]:9}
+LAST_STABLE_VERSION=${LAST_STABLE_VERSION_TAG_ARRAY[0]:1}
 
 RELEASE_TAG=${5-""}
 if [[ -z $RELEASE_TAG ]]; then
-  echo "Path to previous version of aar is not set, using ${LAST_STABLE_VERSION}"
+  echo "Downloading ${LAST_STABLE_VERSION} release from the SDK Registry to check api compatibility with revapi."
   AAR_PATH="s3://mapbox-api-downloads-production/v2/mobile-maps-android-"${MODULE_NAME}"/releases/android/"${LAST_STABLE_VERSION}"/maven/maps-"${MODULE_NAME}"-"${LAST_STABLE_VERSION}".aar"
```

Comment:
if possible I'd prefer not interacting with S3, and use our production endpoint instead, as this is a public repo.

Reply:
Fair, however I'd do it in a separate PR/issue since it seems out of scope for thix fix

Reply:
Replaced with https endpoint